MIP61/MIP61.md

```diff
@@ -93,17 +93,25 @@ The formula is `C*min(1,MKR^q/T^q)`.
+Rates of payment need recalculation only when the amount of delegated MKR changes. A rough approximation is obtained by recalculating only at regular periods (daily or weekly). Governance Facilitators shall look for changes at least daily and include the DAI transfers as part of an Executive Vote once per month.
 
-#### Compensation Eligibility
+It is expected that Recognized Delegates will confirm both the amount of compensation being paid, and the receiving address with the Governance Facilitators in the week leading up to the Executive Vote, if this is not possible for an individual Recognized Delegate, that individual's compensation will not be included in the Executive Vote and instead will be included in a future Executive Vote at a mutually agreeable time.
 
-Governance Facilitators track participation and communication metrics for each Recognized Delegate. In order to receive regular compensation, Recognized Delegates must maintain a minimum percentage on each of these two metrics, namely:
+#### Performance Modifier
 
-- Participation is required to be **at least 90%**.
-- Communication is required to be **at least 90%**.
+Governance Facilitators track participation and communication metrics for each Recognized Delegate. For the purpose of calculating the Performance Modifier, we look at the minimum of the two metrics. For example, if a Recognised Delegate has 85% participation and 80% communication then the Performance Modifier is based on `min(85%, 80%) = 80%`.
+
+- For less than 75%, the Recognised Delegate will receive no compensation.
+- For 75% or above, the Performance Modifier starts at 40% and rises linearly to 100% for metrics of 90% and above.
+
+This is demonstrated in the below graph:
+
+![dc0d6e19d5b85b312de77d11b458a1ecb9ab2e63_2_1035x634|690x422](upload://reIamqwdcWgMAvkMqcjxzhaYqYz.png)
 
 These requirements and metrics may be adjusted in the future depending on the number of Recognized Delegates, the resources available to the Governance Facilitators, and feedback from the delegates themselves.
 
+The intention is that these parameters be considered over the duration of time that a Recognized Delegate is active. The advantage of such an approach is that it allows Recognized Delegates that have been performing at high levels flexibility in terms of unexpected ability to vote, for example, in the event of illness, or planned inactivity, such as a vacation period. If a Recognized Delegate's parameters were calculated on a rolling-basis, for example over three months, taking a week off would have an outsized effect on compensation, particularly if it was a week with a large number of Governance Polls. If it is difficult or uneconomical for Recognized Delegates to take time off when required, there is potential this could lead to burnout and this is consequently undesirable.
```

Comment:
Eh? Are you saying that parameters will not be calculated on a rolling-basis, even a 6 or 12 month moving average? I feel like it would be better to encourage Recognized Delegates to explicitly request time off from start-date to end-date. That way, we know when we're paying them for paid-time-off and we can use a tight 2 or 3 month rolling average on their stats. Just like other Maker employees, we can give them 6 weeks off per year. Thoughts?

Reply:
Actually we should require that Recognized Delegates reserve paid-time-off on a shared calendar to avoid the situation where all delegates go on vacation at the same time.

Reply:
I would separate what from why here. Why can go into the motivation section. I don't think we want to actively manage delegates taking time off. 

I feel like you're misunderstanding the relationship between us and the delegates @jpritikin, we aren't their bosses, MKR Holders are their bosses. We just ensure that we provide information to MKR Holders about their performance, and handle payroll. Not sure where you got '6 weeks off a year' as the standard for Maker employees, there is no standard.

And no, parameters are not calculated on a rolling basis in the above. Delegates can skip up to 10% of the votes with no penalty, and a steadily increasing penalty down to them missing 25% of the votes. Feels like enough leeway.

Reply:
We did discuss a system where a delegate could give notice that they would not be active for a defined period, and this wouldn't count against them - but I'm concerned that may be open to abuse

Reply:
> discuss a system where a delegate could give notice that they would not be active for a defined period

We don't care about individual delegates. We just care that there are enough delegates to vote on polls and secure the hat. I think our scoring system should reflect that.

Reply:
I've moved this to the motivation section

Reply:
Hm, here's a concrete proposal. Calculate the [weighted quantile](https://aakinshin.net/posts/weighted-quantiles/) of the performance modifier. A delegate's MKR weight is the weight. The quantile location will be a parameter set by governance. We could start with 0.5 (i.e., median). If the weighted quantile Performance Modifier is greater than 90% then compensation is not adjusted as described in the `Performance Modifier` section. This allows a delegate to take time off without penalty as long as other delegates are active. If the weighted quantile Performance Modifier is less than 90% then the current rules apply. 

> If a Recognized Delegate's parameters were calculated on a rolling-basis, for example over three months, taking a week off would have an outsized effect on compensation,

Since this would no longer be true, we could look at each month independently. Once things are more automated, we could even apply the same method weekly.

Reply:
Personally I think this is overcomplicating things. Interested in your thoughts @LongForWisdom

Reply:
Gah, you guys are so math adverse. :-)

A simpler approach that gets the same result is to sum the MKR weights for delegates with 90%>= performance stats. If this is more than, say 75%, of the total delegated MKR then compensation is not adjusted as described in the `Performance Modifier` section.

Reply:
Agree that it's overcomplicating things. It's not that we're math adverse, we just don't want to add more complexity to what is already a non-trivial system for compensation.

We should not make a delegate's compensation in any way dependent on the actions of other delegates. Like, this just flat out doesn't work. Gauntlet explicitly does not aim for 100% participation, and it is looking like Hasu won't either. The rules shouldn't change for a delegate because some other delegate is or isn't doing something.

Reply:
OK, I withdraw my idea.